app/controllers/concerns/idv_session.rb

```diff
@@ -6,12 +6,9 @@ def confirm_idv_session_started
   end
 
   def confirm_idv_attempts_allowed
```

Comment:
So, I don't know that we need to change it here in the legacy flow. This code was working. It was only broken in the docauth flow.

Reply:
Not sure that that is accurate because @donjo made it to the phone screen with 5 attempts.  The rationale is that because the old throttling relies on an explicit reset it is inherently prone to locking up.